runtime/CSharp/Antlr4.Runtime/Antlr4.Runtime.csproj

```diff
@@ -158,4 +158,8 @@
     <None Remove="@(Compile)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="@(Antlr4)" />
```

Comment:
:memo: This can be removed after this project updates to a version of ANTLR which provides the functionality automatically.